src/main/java/com/thedariusz/warnme/twitter/TweetDto.java

```diff
@@ -1,66 +1,47 @@
 package com.thedariusz.warnme.twitter;
 
-import java.util.List;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.thedariusz.warnme.twitter.model.Entity;
 
 public class TweetDto {
-
-    private final String tweetId;
-    private final String text;
-    private final AuthorDto author;
-    private final String creationDate;
-    private final List<String> mediaList;
-    private final List<String> hashTags;
-
-    public static TweetDtoBuilder builder() {
-        return new TweetDtoBuilder();
-    }
-
-    private TweetDto(String tweetId, String text, AuthorDto author, String creationDate, List<String> mediaList, List<String> hashTags) {
-        this.tweetId = tweetId;
+    private String id;
+    private String text;
+    private String authorId;
+    private String createdAt;
+    @JsonProperty("entities")
```

Comment:
same here for the annotation. 
btw Are you sure it's just single object if the value from json is `entities` ? maybe it should be list of objects ?

Reply:
I'm sure...
`            "entities": {
                "hashtags": [...
                    ],
                "urls": [
                   ...
                ]
            },
          "id": "1399408696384499718",`